src/edit_python_pe/screens/loading.py

```
@@ -0,0 +1,110 @@
+            _, original_repo = get_repo(self.token)
+            repo_path, forked_repo = fork_repo(self.token, original_repo)
+
+            app = cast("MemberApp", self.app)
+
+            app.original_repo = original_repo
+            app.forked_repo = forked_repo
+            app.repo_path = repo_path
+            app.token = self.token
+
+            self.app.call_from_thread(self.show_success)
+        except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)